app/Http/Api/CoffeeController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Api;
+
+use App\Models\Coffee;
+
+class CoffeeController
+{
+    public function __invoke()
+    {
+        Coffee::create();
```

Comment:
Will coffee be the only statistic we're tracking? Maybe an excuse to use https://github.com/spatie/laravel-stats instead? 😄

Reply:
I was on the fence but decided to keep it vanilla because I'll have to extract the tile to its own package in a minute :)